easybuild/easyblocks/g/gromacs.py

```diff
@@ -324,34 +375,62 @@ def configure_step(self):
                     if not regex.search(out):
                         raise EasyBuildError("Pattern '%s' not found in GROMACS configuration output.", pattern)
 
+    def build_step(self):
+        """
+        Custom build step for GROMACS; Skip if CUDA is enabled and the current
+        iteration is for double precision
+        """
+
+        cuda = get_software_root('CUDA')
+        if cuda:
+            if (not self.cfg.get('double_precision') and
```

Comment:
I don't understand this condition: If `double_precision` is None or False and it is enabled by configopts? Also what about the configure option? (`--enable-double` IIRC)

Reply:
CUDA builds are only available from GROMACS 4.6 and forward
--enable-double is only uesd for < 4.6 builds.